migrations/versions/rev-2021-01-06-23:25:53-e3f69cc43643_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
+
+Revision ID: e3f69cc43643
+Revises: 2d0760003a8a
+Create Date: 2021-01-06 23:25:53.240369
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'e3f69cc43643'
+down_revision = '2d0760003a8a'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('sessions_speakers_links', 'session_state')
+    op.add_column('speaker', sa.Column('rank', sa.Integer(), nullable=False))
```

Comment:
server default missing. Did you try to run the migrations?

Reply:
Yes, I don't know why it's missing.

Reply:
You have to manually add it